clarity_ext/context.py

```diff
@@ -124,6 +131,17 @@ def output_result_file_by_id(self, file_id):
     def output_result_files(self):
         return self.artifact_service.all_output_files()
 
+    @property
+    def user_initials(self):
+        return self.current_user.initials
+
+    @property
+    def pid(self):
+        return self.session.current_step_id
+
+    def today(self, format):
```

Comment:
Yes, I was considering that fact when writing. I had in mind to get as simple as possible in the scripts, but I think it's maybe better with the suggestion above. I'll change it before the merge.